### DISCUSSION TITLE
Turn of screen using custom code

### DISCUSSION THREAD

**Discussion by sbruynin** [HASwitchPlate/openHASP#816] [Q&A] [answered]
I want to turn of the screen using custom code. (Not via mqtt to set backlight off !)
I tried "lv_disp_set_brightness" but that seems not to be supported.
What is the recommended command to put in my custom code?

**Comment by fvanroie** [comment, marked as answer]
The backlight is not LVGL controlled, so that's a HASP command. All HASP commands are processed in `hasp_dispatch.cpp`
Try `dispatch_backlight("backlight", "off", TAG_CUSTOM);`

**Reply by sbruynin** [reply]
Thanks for the quick reply. The above command does the job.

**Reply by RaW79** [reply]
Is this a good way to copy this command to e.g. the system script idle_long.cmd to turn of the display. I´m not using HA so do not have this configuration option in the HA module to configure this.  Simply like to have a auto off after some seconds which seems to be usable by idle_short and idle_long configuration in the device setup.

**Reply by fvanroie** [reply]
Yes you can use the backlight command in scripts, however this topic was specific to custom code control.
